sdk/include/ddk/indicml.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * PROJECT:     ReactOS DDK headers
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Indicator (internat.exe) window class
+ * COPYRIGHT:   Copyright 2025 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
```

Comment:
I would suggest you to move this file outside of ddk and put it in an "ime" subdirectory in include instead. (Like the imm.h file by the way.)
Indeed this file is part of a sample (e.g. `C:\WINDDK\3790.1830\src\ime\inc` or `C:\WINDDK\NTDDK\src\ime\inc`) in the ddk, but is not within the ddk include subdirectory.

Reply:
> put it in an "ime" subdirectory in include instead. (Like the imm.h file by the way.)

That brain memory contains some mistake. `<imm.h>` is in the `psdk` folder.
`<immdev.h>` is in the `ddk` folder.

Reply:
ah so it's more spread than I thought ^^ in any case, "indicml.h" in Windows DDK is _not_ in the includes directory, but in some sample, hence it would be better to have them somewhere else.

Reply:
Moved.